compiler/rustc_codegen_cranelift/src/lib.rs

```diff
@@ -196,7 +192,10 @@ impl CodegenBackend for CraneliftCodegenBackend {
             }
         } else {
             vec![]
-        }
+        };
+        // FIXME do `unstable_target_features` properly
+        let unstable_target_features = target_features.clone();
```

Comment:
This is not just unstable target features as the name of this variable implies, but all including unstable ones. Maybe rename it?

Reply:
`all_target_features` and `stable_target_features` would be clearer names, yes. But `Session` already has fields called `unstable_target_features` and `target_features`, and this is the code that is used to set those fields, so I used those names for consistency. If you think it's important, maybe it should be a follow-up.